app.js

```diff
@@ -0,0 +1,12 @@
+class Calculator{
```

Comment:
`add()` and `multiply()` methods are not fully cover some interesting cases like passing not a number values to it.